packages/expo-location/src/Location.types.ts

```diff
@@ -105,7 +105,8 @@ export type LocationOptions = {
   /**
    * Location manager accuracy. Pass one of `Accuracy` enum values.
    * For low-accuracies the implementation can avoid geolocation providers
-   * that consume a significant amount of power (such as GPS).
+   * that consume a significant amount of power (such as GPS). Default is
+   * LocationAccuracy.Balanced
```

Comment:
```suggestion
   * that consume a significant amount of power (such as GPS).
   *
   * @default LocationAccuracy.Balanced
```
We have a special annotation that specifies the default value.